charts/nats/templates/jetstream-ssl-conf.yaml

```diff
@@ -0,0 +1,131 @@
+
+########################################
+## Nats Jetstream SSL Certificate   ####
+########################################
+{{ if and  .Values.global.nats.jetStream.enabled .Values.global.nats.jetStream.tls }}
+{{- $dn := printf "%s-nats" .Release.Name }}
+{{- $ns := printf "%s" .Release.Namespace }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "nats.name" . }}-tls-script-config
+  labels:
+    app: {{ template "nats.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+  annotations:
+    "helm.sh/hook": "pre-install,pre-upgrade"
+    "helm.sh/hook-weight": "-2"
+data:
+  tls-script.sh: |
+
+    #!/bin/bash
+    export DOMAIN={{ .Release.Name }}-nats
+    export RELEASE_NAME={{ .Release.Namespace }}
+
+    # Create private CA
+    openssl genrsa -out rootCA.key 2048
+    openssl req -x509 -new -nodes -key rootCA.key -sha256 -subj "/CN=ASTRONOMERCA" -days 365 -out rootCA.pem
+
+    cat > server_config.cnf <<EOF
+    [req]
+    default_bits = 2048
+    prompt = no
+    default_md = sha256
+    distinguished_name = dn
+    req_extensions = req_ext
+
+
+    [dn]
+    C = US
+    ST = WA
+    L = Seattle
+    O = ${DOMAIN}
+    OU = ${DOMAIN}
+    CN = ${DOMAIN}.astronomer.svc.cluster.local
+
+    [ req_ext ]
+    basicConstraints = CA:FALSE
+    keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+    subjectAltName = @alternate_names
+
+    [alternate_names]
+    DNS.1 = ${DOMAIN}
+    DNS.2 = ${DOMAIN}.${RELEASE_NAME}
+    DNS.3 = ${DOMAIN}.${RELEASE_NAME}.svc
+    DNS.4 = *.${DOMAIN}
+    DNS.5 = *.${DOMAIN}.${RELEASE_NAME}
+    DNS.6 = *.${DOMAIN}.${RELEASE_NAME}.svc
+    DNS.10 = {{ .Values.global.baseDomain }}
+    DNS.8 = localhost
```

Comment:
Why does this list go DNS.{1..6}, DNS.10, DNS.8 ?

Reply:
will trim this down